simulator/CMakeLists.txt

```diff
@@ -35,6 +35,7 @@ set(CPPS infra/macro_test.cpp
     modules/decode/decode.cpp
     modules/execute/execute.cpp
     modules/mem/mem.cpp
+    modules/prediction/prediction.cpp
```

Comment:
The name is misleading — it does not predict anything. I suggest to rename it to `branch`, including classes and objects.